libs/wingsdk/src/platform/platform.ts

```diff
@@ -12,6 +12,11 @@ export interface IPlatform {
    */
   readonly target: string;
 
+  /**
+   * The schema for the parameters the platform accepts
+   */
+  readonly parameters?: any;
```

Comment:
I want to type this using @types/json-schema but having jsii issues (still looking into it)